mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -133,15 +144,21 @@ private object ScalaConverter {
     user = event.getUser,
     path = DTO.MailboxPath.fromJava(event.getMailboxPath),
     mailboxId = event.getMailboxId,
-    added = event.getAdded.asScala.map(entry => entry._1 -> MetaDataDTO.MessageMetaData.fromJava(entry._2)).toMap
-  )
+    added = event.getAdded.asScala.map(entry => entry._1 -> MetaDataDTO.MessageMetaData.fromJava(entry._2)).toMap)
+
+  private def toScala(event: JavaMessageMoveEvent): DTO.MessageMoveEvent = DTO.MessageMoveEvent(
+    user = event.getUser,
+    previousMailboxIds = event.getMessageMoves.getPreviousMailboxIds.asList().asScala.toList,
```

Comment:
i bet you could remove `asList()` without hurting the code :)

Reply:
Good catch